cachito/workers/config.py

```diff
@@ -66,10 +67,7 @@ class Config(object):
     # Route gomod tasks and npm tasks to separate queues. This is useful if workers are dedicated
     # to specific package managers.
     task_routes = {
-        "cachito.workers.tasks.gomod.*": {
```

Comment:
It's strange that this was not needed before. Could you please move this and the formatting change below to a separate commit?

The reason is that this change is not related to the strict gomod vendor feature. This makes the commit less concise and it also makes it difficult to revert if we ever need to since then we'd need to fix the formatting separately.